__init__.py

```diff
@@ -334,5 +354,13 @@ def unregister():
 	del bpy.types.Scene.exportHiddenGeometry
 	del bpy.types.Scene.blenderF3DScale
 
+	del bpy.types.Scene.fast64
+
 	for cls in classes:
 		unregister_class(cls)
+	
+	bpy.app.handlers.load_post = [
+		handler for handler in bpy.app.handlers.load_post
+		if handler is not after_load
+	]
```

Comment:
I'd use remove() instead of filtering the list like this, I'm surprised this even works tbh (usually I'm careful to not assume blender uses python data structures since it's just exposing C most of the time)

Reply:
makes sense, done